packages/parser-manifest/src/parser.ts

```diff
@@ -125,7 +126,16 @@ export default class ManifestParser extends Parser<ManifestEvents> {
         });
     }
 
-    private async validateManifest (fetchEnd: FetchEnd) {
+    private async validateManifest(fetchEnd: FetchEnd) {
+        /*
+         * Chrome fetch the manifest automatically so we need to
+         * ensure that the validation is done just once.
+         */
+        if (this.parsed) {
+            return;
+        }
+
+        this.parsed = true;
```

Comment:
Make sure to reset `this.parsed` on `scan::end` or errors won't get reported from `connector-local` on re-runs (e.g. when run from `extension-vscode` or the CLI in `watch` mode).

This isn't the first time I've seen cached data cause a problem so I filed #1665 to track enhancing `utils-test-helpers` to automatically test re-runs for all hints.

Reply:
done!